src/server/cmd/pachd/main.go

```diff
@@ -103,6 +103,7 @@ type appEnv struct {
 	ImagePullSecret       string `env:"IMAGE_PULL_SECRET,default="`
 	NoExposeDockerSocket  bool   `env:"NO_EXPOSE_DOCKER_SOCKET,default=false"`
 	ExposeObjectAPI       bool   `env:"EXPOSE_OBJECT_API,default=false"`
+	MemoryRequest         string `env:"PACHD_MEMORY_REQUEST,default=1T"`
```

Comment:
Hang on a sec, `1T` default memory request?

Reply:
Go big or go home. Jk, so that actually does make sense in the context of how it is being used. Maybe the field name could use some work though. This field is used for the memory limiter in the driver to prevent OOM kills when users are doing big put file splits. The value is pulled from the container memory request, and if that is not available, then we pretty much set no limit (thus the 1T).

Reply:
> Go big or go home
lol

That makes sense, is there a way we could have it be `0` and have that indicate no limit at all? Or does it need to be a real value? Not really a big deal, just would be a tiny bit cleaner.

Reply:
So, I think the 1T solution is the cleanest because I do not need to special case everywhere I use the memory limiter. We could still special case on 0, but that means we are either setting the value read by the driver to a high value or each memory limiter call would need to check for this special case.

Reply:
I'm on board, go big or go home.